BLL/DrinkLoader.cs

```diff
@@ -9,26 +10,30 @@ namespace BLL
 {
     public class DrinkLoader
     {
-        public List<Drink> InitializeDrinksFromFile()
+        public FileLoaderDto InitializeDrinksFromFile()
```

Comment:
Please revisit this again and make sure that entire solution is building correctly without errors. As we talked probably there is no point now in returning DTO by this method, since ingredients are part of the drink entity now.